_posts/2019-08-13-how-to-deploy-production-grade-vpc-aws.adoc

```diff
@@ -710,6 +715,8 @@ module "vpc" {
 }
 ----
 
+# TODO: This pattern of linking to the actual code in a repo for the larger files, we should adopt this in the
+# landing zone guide!
```

Comment:
We can't do this for landing zone because the acme repo does not use landing zone. And since the acme repos will be retired once service catalog reaches GA, we actually will need to update this...

On another note, we need to discuss what perspective these guides should be written in going forward. The main question to answer will be: are these guides for building the architecture catalog from scratch (and thus are based on the Service Catalog code)? Or are we assuming the user will also create a service catalog from scratch (which is what it is now)?

Reply:
Yeah, for that last point, I think the latter is better. But, I think for now, we should update this to work as it is, the former way.

If our users have access to the SC, we should update the guides to link to the SC code for each of the code snippet examples. E.g., we should link to the examples folder in aws-service-catalog, for each of the TF files, and in the guide, we should explain what those snippets do, because we want to encourage understanding (not just following instructions). Therefore, we should still have the walkthrough. It would be great to link to the github snippets which are external to this repo. I don't know if that's possible! I looked into it a couple of days ago and didn't see how it could be done.

I don't see the benefit to building the SC from scratch, especially if it means immediate code-drift and constant maintenance of this guide as the SC (and/or modules) changes.

Reply:
That seems like a good approach!